service_container/service_decoration.rst

```diff
@@ -52,7 +52,33 @@ the original service is lost:
 Most of the time, that's exactly what you want to do. But sometimes,
 you might want to decorate the old one instead. In this case, the
 old service should be kept around to be able to reference it in the
-new one. This configuration replaces ``app.mailer`` with a new one, but keeps
+new one.
+
+Suppose you want to process the text parts to send behind the ``app.mailer``, you will create a mailer decorator::
+
+    // AppBundle/DecoratingMailer.php
+    namespace AppBundle\DecoratingMailer;
```

Comment:
If we were to add a namespace I think we should use `Mailer`. But then other examples would need an update too.
What do other thinks?